system-test/testnet-performance/upload_results_to_slack.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+upload_results() {
```

Comment:
please use a more descriptive name as this function will be inserted into the namespace of the called.  "upload_results" to where?  `upload_results_to_slack` perhaps